javascript-bank/bank.js

```diff
@@ -0,0 +1,54 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  //
+  this.balance = balance;
+  this.holder = holder;
```

Comment:
This is still creating additional properties on the object. Try console logging out `this` in each of your functions, you will start to see these extra properties start to bulk up the object whenever these functions are called. This isn't data that needs to continue to exist after this function runs, so it shouldn't be a property on the object, just a local variable to the function.